index.html

```diff
@@ -4975,6 +4975,646 @@ <h2 id="AudioWorkerParamDescriptor">
+                <a>AudioWorkletNode</a> to create instances of an
+                <a>AudioParam</a> in the node.
+              </p>
+            </dd>
+            <dt>
+              readonly attribute AudioContextInfo contextInfo
+            </dt>
+            <dd>
+              <p>
+                This getter returns an <a><code>AudioContextInfo</code></a>
+                object that describes the associated <a>AudioContext</a>.
+              </p>
+            </dd>
+            <dt>
+              attribute EventHandler onmessage
+            </dt>
+            <dd>
+              <p>
+                The <code>onmessage</code> handler is called whenever the
+                <a><code>AudioWorkletNode</code></a> posts a message back to
+                the corresponding processor object.
+              </p>
+            </dd>
+            <dt>
+              void process()
+            </dt>
+            <dd>
+              <p>
+                This method gets executed isochronously by the audio
+                <a>rendering thread</a>.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  Float32Array[][] inputs
+                </dt>
+                <dd>
+                  <code>inputs[n][m]</code> is a <code>Float32Array</code> of
+                  audio samples for the <code>m</code>th channel of
+                  <code>n</code>th input. While the number of inputs is fixed
+                  at the construction, the number of channels can be changed
+                  dynamically.
+                </dd>
+                <dt>
+                  Float32Array[][] outputs
+                </dt>
+                <dd>
+                  <code>outputs[n][m]</code> is a Float32Array object
+                  containing the audio samples for <code>m</code>th channel of
+                  <code>n</code>th output.
+                </dd>
+                <dt>
+                  Object parameters
+                </dt>
+                <dd>
+                  Provides an association between the names of the
+                  <a><code>AudioParam</code></a>s specified by the
+                  <a><code>AudioParamDescriptor</code></a>s during construction
+                  and the automation values for the
+                  <a><code>AudioParam</code></a>. Thus,
+                  <code>parameters["name"]</code> returns a
+                  <code>Float32Array</code> of the automation values
+                  corresponding to the <a><code>AudioParam</code></a> named
+                  <code>"name"</code>.
+                </dd>
+              </dl>
+            </dd>
+            <dt>
+              void postMessage()
+            </dt>
+            <dd>
+              <p>
+                It may be called to send a message to the
+                <a><code>AudioWorkletNode</code></a>, via the algorithm defined
+                by the Worker specification.
```

Comment:
We don't have a traditional event loop inside the rendering thread, so the definition in the Worker specification does not work, since it relies on having an event loop. We have something very similar, but maybe a few adaptations have to be made.